bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java

```diff
@@ -983,4 +983,25 @@ public static String escapeHtml(@Nullable String text) {
         }
         return grouped;
     }
+    
+    public static String toASCIILetterLowerCase(String str) {
+        if (str != null && !str.trim().isEmpty()) {
+            char[] chars = str.toCharArray();
+            int i = 0;
+            for (char c : chars) {
```

Comment:
Let's use `for (i = 0; i < chars.length; i++)` here and retrieve `c` by index

Reply:
Done!

Reply:
I can't see the changes 🤔

Reply:
@ShadelessFox 
Apologies. Missed pushing these changes. Pls check now.